style.css

```diff
@@ -16,6 +16,67 @@ ul {
   padding: 0;
 }
 
+.el-order {
+  order: 0;
+}
+
+.el-order1 {
+  order: 1;
+}
+
+.about-mb {
+  margin-bottom: -115px;
+}
+
+.fbuttom-mt {
+  margin-top: 0;
+}
+
+.overlay {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 0;
+  left: 0;
+  top: 0;
+  background: #6070ff;
+  background-color: rgba(96, 112, 255, 0.902);
+  overflow-x: hidden;
+  transition: 0.5s;
+}
```

Comment:
- [ ] Kindly make your background match the one in the Figma by making the background blur.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/179550636-da8f51ba-08aa-4b22-8916-af1df9839ceb.png)  | ![image](https://user-images.githubusercontent.com/53402600/179550722-1aa655b7-12c0-4f13-ab53-21b874eddb82.png)|